### ISSUE TITLE
Error with long time upload

### ISSUE THREAD

**Issue by MrMusic** [JoomGalleryfriends/JoomGallery#14]
When a long time upload (Refresher) occurs an error.
The cause is probably a code change in the upload.php here: https://github.com/JoomGalleryfriends/JoomGallery/pull/8/commits/f567fb66f454a588c37417f8b81b9b21d78aeac3
If I change the line 2224 in the current 3.4.0 branch so it will work again:
```
if(isset($new_info['imgtext']))
{
  $new_info['imgtext'] = JComponentHelper::filterText($this->_mainframe->input->post->get('imgtext', '', 'raw'));
}
```
Maybe there is a better solution?

**Comment by Erftralle** [comment]
I can confirm the bug. Please open a pull request with the fix so that it can be tested.

**Comment by Erftralle** [comment]
Closed as we have an already merged pull request #15 providing a fix for this issue.
